### ISSUE TITLE
Lacunar halos around chondrocytes in skeletal cartilage

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5093]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

https://www.ncbi.nlm.nih.gov/pubmed/8571951
**Parent term (use hpo.jax.org/app)**
Consider new subontology for abnormal chondrocyte morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/256050
**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Examination of resting cartilage showed the presence of concentric fibrillar rings surrounding chondrocytes with a faintly fibrillar matrix between cells. These concentric rings, or “lacunar halos,” were PAS positive and stained intensely black with the methenamine silver nitrate-alcian blue (See Figure 4a,b of PMID:3799721).

Lacunar halos, formed by multiple concentric rings around chondrocytes of resting cartilage, were a distinctive feature in two patients that were examined. The rings forming lacunar halos were
observed with eosin or PAS but were seen to best advantage with silver methenamine stain.Identification of lacunar halos in the cartilage of infants with the de la Chapelle dysplasia provides a striking and diagnostically helpful marker. Their presence in the cartilage of infants with achondrogenesis is also of interest, even though these disorders are readily distinguished by radiographic criteria, insofar as further characterization may offer insight into the molecular
defects.

**Comment by pnrobinson** [comment]
Abn chondrocyte morphology
def: Abnormal structure of chondrocytes, mesodermal cells that make up cartilage. Chondrocytes in joint (articular) cartilage normally do not divide after skeletal maturity. Chondrocytes in the epiphyseal growth plate differentiate in the course of endochondral ossification, after which chondrocytes undergo apoptosis or become osteoblasts.
